plugins/network-elements/nuage-vsp/src/com/cloud/agent/api/StartupVspCommand.java

```diff
@@ -26,4 +26,14 @@
     public StartupVspCommand() {
         super(Host.Type.L2Networking);
     }
+
+    @Override
+    public boolean equals(Object obj) {
+        return super.equals(obj);
+    }
+
+    @Override
+    public int hashCode() {
+        return super.hashCode();
+    }
```

Comment:
We can't, the <code>toString()</code> method is made <code>final</code> in the base class <code>Command</code>